src/main/java/com/mraof/minestuck/tileentity/CassettePlayerTileEntity.java

```diff
@@ -0,0 +1,44 @@
+package com.mraof.minestuck.tileentity;
+
+import net.minecraft.inventory.IClearable;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.tileentity.TileEntity;
+
+public class CassettePlayerTileEntity extends TileEntity implements IClearable {
```

Comment:
While not strictly neccessary, I'd advice relocating some functionality regarding cassettes to be relocated to this class, primarily cassettes being played. Feel free to do something or nothing about it at your own discretion.